flag.go

```diff
@@ -420,6 +491,51 @@ func (f IntFlag) GetName() string {
 	return f.Name
 }
 
+// Int64Flag is a flag that takes a 64-bit integer
+// Errors if the value provided cannot be parsed
```

Comment:
TRIVIAL: I don't think the second line of this comment is truthful, so it should probably be removed from the other flag type comments as well (for later).